mips/pmap.c

```diff
@@ -347,7 +348,7 @@ void tlb_exception_handler(exc_frame_t *frame) {
       log("Address %08lx not mapped by any active pmap!", orig_vaddr);
       goto fault;
     }
-    if (is_valid(pmap->pde[index])) {
+    if (pmap->pde && is_valid(pmap->pde[index])) {
```

Comment:
I expect this issue will never appear unless kernel misbehaves, so it does not affect this branch much. And since there are major changes to `pmap` coming soon anyway, I believe this PR is not a right place for considering this hypothesized fix - so I'll revert it for now, and possibly discuss it as a separate feature.